content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,117 @@
+# Capacity Planning
+
+This section describes some of the challenges related to scaling `rippled` servers for testing and production deployments. Additionally, it describes how hardware and configuration settings relate to those challenges. Finally, this section offers recommendations to assist you in properly setting up `rippled` to meet the goals for the use-case of your deployment.
+
+Even the most minimally functional `rippled` server must contain the most recently validated ledger versions to submit transactions to the network and verify the integrity of the XRP Ledger. Beyond these requirements, consider the following possible business requirements:
```

Comment:
> Beyond these requirements, consider the following possible business requirements:

Maybe use "business needs" to avoid using "requirements" twice?